tools/migration/db/util/mysql_pgsql_1_5_0.sh

```diff
@@ -79,4 +79,38 @@ EOF
         stop_mysql root
         stop_pgsql $1 
     fi
+}
+
+function up_clair {
+    # clair DB info: user: 'postgres' database: 'postgres'
+
+    set +e
+    if [[ $(psql -U $1 -d postgres -t -c "select count(*) from vulnerability;") -eq 0 ]]; then
+        echo "no vulnerability data needs to be updated."
+        return 0
+    else
+    
+        ## it's not a clean clair db, so cannot execute the import step.
+        ## fail at here to call user to clean DB, then to run clair db migration.
+        if [[ $(psql -U $1 -d postgres -t -c "select count(*) from pg_tables where schemaname='public';") -ne 0 ]]; then
```

Comment:
This should be checked when pgsql is running with new directory as the data directory?